pkg/hub/csr/controller.go

```diff
@@ -158,16 +159,20 @@ func isSpokeClusterClientCertRenewal(csr *certificatesv1beta1.CertificateSigning
 		return false
 	}
 
-	if len(x509cr.Subject.Organization) != 1 {
+	requestingOrgs := sets.NewString(x509cr.Subject.Organization...)
+	if requestingOrgs.Has(user.SpokeClustersGroup) { // optional common group for backward-compatibility
```

Comment:
would you add some ut for this?

Reply:
test cases added covering the latest and the previous format of CSR